test/spec.yml

```diff
@@ -12,6 +12,7 @@
                 bn deploy <function> [options]            Deploys a function to the cloud
                 bn remove <function> [options]            Remove a previously deployed function
                 bn invoke <function> [options]            Invoke a Binaris function
+                bn perf <function> [options]              Measure invocation latency (experimental)
```

Comment:
the spec should also have an actual use of this command

Reply:
#293